### ISSUE TITLE
[A11y] Desktop/Mobile Work Show Page-CPD Child: All 5 image thumbnail labels

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1953]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Work Show

### Admin user?

No

### Issue Description

Work Show Page-CPD Child: https://staging.oregondigital.org/concern/documents/df727057n?locale=en

All five thumbnails show the following failed instances for labels:
label: ensures every form element has a label

Path: 
#batch_document_df727084j

Snippet:
`<input type="checkbox" onfocus="setGalleryCheckbox(this)" onblur="setGalleryCheckbox(this)" onchange="setGalleryCheckbox(this)" name="batch_document_ids[]" id="batch_document_df727084j" value="df727084j" class="batch_document_selector">`

### Suggested Fix

Fix ONE of the following:

- Form element does not have an implicit (wrapped) <label>
- Form element does not have an explicit <label>
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute
- Element has no placeholder attribute
- Element's default semantics were not overridden with role="none" or role="presentation"

### Can anybody QA or does this need an expert?

A11y familiarity recommended

**Comment by CGillen** [comment]
Fixed?

**Comment by azle** [comment]
I revalidated the page and has 1 error, see screen snap:

<img width="811" alt="Screen Shot 2022-02-02 at 8 54 42 AM" src="https://user-images.githubusercontent.com/8353860/152199940-c0795e6d-00b0-4a75-acdf-09a5346e6128.png">

**Comment by CGillen** [comment]
Above issue is related to an UV issue we currently can not fix. Please disregard until we can edit the UV iframe (if we decide to)
